docs/proposals/localDisk_volume.md

```diff
@@ -0,0 +1,309 @@
+# localDisk volume
+
+## Abstract
+
+This is a proposal for implementing a new volume type, localDisk, which will support,
+
+ 1. the Node has multiple physical disks and pod requires 1 ~ N disks as its storage volumes.
+ 2. the Node has lvm VG and pods allocate specific size LVs from this VG.
+ 3. When pods are deleted, the data in the volume could be either retained or deleted.
+ 
+As localDisk volume life cycle would be different from pod, we need volume APIs to do the management. 
+The existing PersistentVolume/PersistentVolumeClaim(PV/PVC) API is for network storage in cluster, we propose new LocalVolume/LocalVolumeClaim(LV/LVC) API to manage localDisk volume.                      
+
+## Motivation
+
+* This new volume type is for application that requires high I/O performance, one pod need a dedicated disk.
+* The bigdata application, e.g. Kafka, Cassandra, Hadoop can deal with a bunch of distinct volumes, one pod requires multiple disks.  
+* The emptyDir and hostPath volume doesn't have disk size limitation, the localDisk volume is able to provide this feature by using lvm.

```

Comment:
we can enforce quota on directories (eg: http://souptonuts.sourceforge.net/quota_tutorial.html) the goal here should be isolation, creating a logical volume is just one way to achieve that IMO. In fact we don't care how it is achieved, as long as it enforces the specified quota.